Sources/NIOOpenSSL/ByteBufferBIO.swift

```diff
@@ -313,7 +313,7 @@ final class ByteBufferBIO {
             assert(!self.mustClearOutboundBuffer)
         }
 
-        let writtenBytes = self.outboundBuffer.write(bytes: buffer)
+        let writtenBytes = self.outboundBuffer.writeBytes(buffer)
```

Comment:
@tanner0101 this should `import _NIO1APIShims` so it doesn't get broken all the time

Reply:
done: https://github.com/apple/swift-nio-ssl/pull/60/commits/559bda3d084ac60d606e7478393a4f1fa2acb8ed